frontend/src/components/ListItems/list-items.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
This doesn't actually test list-items, so we should note that somewhere or let this test fail for now

Reply:
None of my tests do anything for now. I feel like letting them fail is enough of a reminder.

Reply:
Okay, sounds good.